lib/ts/controllers/s-expandable-control.ts

```diff
@@ -145,12 +147,13 @@
             // attribute; for checkable controls this also means setting the `is-collapsed` class
             this.element.setAttribute("aria-expanded", this.isCollapsed() ? "false" : "true");
             if (this.isCheckable) {
-                var cc = this.controlledCollapsible;
+                var cc = this.controlledCollapsibles;
                 if (cc) {
                     var expected = !this.isCollapsed();
-                    var actual = cc.classList.contains("is-expanded");
-                    if (expected !== actual) {
-                        cc.classList.toggle("is-expanded", expected);
+                    if (cc.some(element => element.classList.contains("is-expanded") !== expected)) {
```

Comment:
This looks fine to me, but the inconsistent usage of some (here) vs every (isCollapsed) messed with my brain a little, so I added some comments. Commit as desired:

 ```suggestion
                    // if any element does not match the expected state, set them all to the expected state
                    if (cc.some(element => element.classList.contains("is-expanded") !== expected)) {
```